easybuild/easyconfigs/g/gdc-client/gdc-client-1.3.0-foss-2018b-Python-2.7.15.eb

```diff
@@ -0,0 +1,134 @@
+easyblock = 'Bundle'
+
+name = 'gdc-client'
+version = '1.3.0'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://gdc.nci.nih.gov/access-data/gdc-data-transfer-tool'
+description = """The gdc-client provides several convenience functions over the GDC API which provides general
+ download/upload via HTTPS."""
+
+toolchain = {'name': 'foss', 'version': '2018b'}
+
+# this is a bundle of Python packages
+exts_defaultclass = 'PythonPackage'
```

Comment:
@fizwit Please enable both `download_dep_fail` and `use_pip` for the extensions being installed, as follows:

```python
exts_default_options = {
    'download_dep_fail': True,
    'use_pip': True,
}
```